src/main/java/kr/motd/maven/os/SystemPropertyActionFacade.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 Trustin Heuiseung Lee.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kr.motd.maven.os;
+
+/**
+ * Interface exposing system property operations.
+ */
+public interface SystemPropertyActionFacade {
```

Comment:
I find it weird to call an interface a façade. And "Action" in the name is weird as well; it's unclear to me what purpose it serves. Maybe name it something more like `SystemPropertyProvider` and `FileProvider`?

Reply:
Okay, an interface already functions as a facade. 

`SystemPropertyOperationProvider` and `FileOperationProvider` seem to be better.